src/browser/lib/movementHandler.js

```diff
@@ -40,9 +40,12 @@ export default function movementHandler(BaseComponent) {
     }
 
     handleKeyUp(e) {
-      if (e.keyCode === 13) {
+      if (e.keyCode === 13 || e.keyCode === 9 || e.keyCode === 40) {
```

Comment:
It is convenient to put key codes into constans like `ENTER_KEY`, `ARROW_DOWN_KEY`, ...